### PR TITLE
css and connection fix

### PR DESCRIPTION
**Pull request by gsdstr**
use mongoose.connection
update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <img src="http://vuejs.org/images/logo.png" height="50"> <img src="https://coligo.io/images/express.svg" height="50"> <img src="https://upload.wikimedia.org/wikipedia/en/thumb/4/45/MongoDB-Logo.svg/527px-MongoDB-Logo.svg.png" height="50"> <img src="https://worldvectorlogo.com/logos/nodejs-icon.svg" height="50"> <img src="https://camo.githubusercontent.com/66747a6e05a799aec9c6e04a3e721ca567748e8b/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f313336353838312f313931383337332f32653035373166612d376462632d313165332d383436352d3839356632393164343366652e706e67" height="50">
 
 [![Known Vulnerabilities](https://snyk.io/test/github/icebob/vue-express-mongo-boilerplate/badge.svg)](https://snyk.io/test/github/icebob/vue-express-mongo-boilerplate)
-![Node 6](https://img.shields.io/badge/node-6.x.x-green.svg)
+![Node 6](https://img.shields.io/badge/node-10.x.x-green.svg)
```
Comment (icebob) [inline README.md]: Please update the caption too.

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
 ![VueJS 2](https://img.shields.io/badge/vuejs-2.3.x-green.svg)
-![Webpack 2](https://img.shields.io/badge/webpack-2.6.x-green.svg)
+![Webpack 2](https://img.shields.io/badge/webpack-4.17.x-green.svg)
```
Comment (icebob) [inline README.md]: Please update the caption too.

**Comment by gsdstr** [comment]
Updated

**Comment by icebob** [comment]
Thanks.

**Pull request merged**